### PR TITLE
Fix/slider

### PR DESCRIPTION
**Pull request by boranfurkan**
- Description:
- ClickUp links:
- What are the acceptance criteria? What are the steps to test that this code is working?

In this pull request I have resolved the problem with current slider. I have created two different versions of slider by using radix-ui libraries slider component. I have also replaced the old slider in secondary question answer place. 

Click up link: https://app.clickup.com/t/86byhc3wa

### CODE REVIEW

**Comment by marvinmarnold** [comment]
@boranfurkan your build is failing. Do `yarn build` to debug locally.

File: app/components/PrimarySlider/PrimarySliderV2.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+interface PrimarySliderV2Props {
+  value: number;
+  setValue?: (value: number) => void;
+  backgroundColor?: string;
+  progressColor?: string;
+  hideThumb?: boolean;
+  className?: string;
+}
+
+const PrimarySliderV2 = ({
```
Comment (marvinmarnold) [inline app/components/PrimarySlider/PrimarySliderV2.tsx:13]: Can you rename PrimarySliderV2 or add comments to make it more clear how it's different from PrimarySlider?

File: app/components/PrimarySlider/PrimarySlider.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <Slider.Root
+      className={`relative flex items-center select-none touch-none w-full min-h-5 ${className}`}
+      defaultValue={[50]}
+      max={100}
+      step={1}
+      onValueChange={(value) => setValue && setValue(Number(value))}
+      value={[value]}
+    >
+      <Slider.Track
+        className={`relative grow rounded-[10px] h-[36px] overflow-hidden ${backgroundColor ? backgroundColor : "bg-pink"}`}
```
Comment (marvinmarnold) [inline app/components/PrimarySlider/PrimarySlider.tsx:31]: Can also do `backgroundColor ?? "bg-pink"` to be more concise.

Comment (AnteVuletic) [reply app/components/PrimarySlider/PrimarySlider.tsx:31]: We have classNames package to handle conditional css classes, please look into it for consistency.

**Comment by AnteVuletic** [comment]
Can we add usage to storybook for the new slider?

File: app/components/PrimarySlider/PrimarySlider.tsx (hunk 2 above)
Comment (boranfurkan) [reply app/components/PrimarySlider/PrimarySlider.tsx:31]: Chomp

Re: Can you rename PrimarySliderV2 or add comments to make it more clear how it's different from PrimarySlider?

Here as you can see I have created new and the old versions of the slider from the figma. I will comment out it in the file as well but I guess would be better to declare the version of the slider in the name to not mix them up.

Comment (boranfurkan) [reply app/components/PrimarySlider/PrimarySlider.tsx]: @marvinmarnold  @AnteVuletic I have realised that TrueFalse scale component was being used in 2 different places with different purposes. 

image
1-) To get users answer for true false questions. 

image
2-) To show the results by updating the same component with classnames etc to show results. 

I personally do not find this approach clean. The code file was way too messy already. If @marvinmarnold want me to create these two as separate components I can create it. Also I have resolved the build errors now. Ready for your review again.

**Review by marvinmarnold — APPROVED** [review]

**Pull request merged**